.gitlab-ci.yml

```diff
@@ -198,6 +198,7 @@ test-lin-dev-clang-cran:
     - echo 'CFLAGS=-g -O2 -fno-common -Wall -Wvla -pedantic -fstack-protector-strong -D_FORTIFY_SOURCE=2' > ~/.R/Makevars
     - echo 'CXXFLAGS=-g -O2 -fno-common -Wall -Wvla -pedantic -fstack-protector-strong -D_FORTIFY_SOURCE=2' >> ~/.R/Makevars
     - *install-deps
+    - clang-tidy -extra-arg=-I/usr/local/lib/R/include -checks='readability-inconsistent-declaration-parameter' src/*.c -- -std=c11
```

Comment:
what will this look like for manually specifying, say, dozens of checks? Will it just be

```shell
clang-tidy -extra-arg=... \
  -checks='...' \
  src/*.c
```

Is it possible to keep this as a list in a file (say in .ci)?

(doesn't need to be done in this PR since we only have one check thus far, just trying to think ahead before it gets much harder to refactor)

Reply:
Yes. We can also globbing for checks e.g. `-check='readability-*'` or multiple checks with `-checks='bugprone-*,readability-*'`